ios/Job.swift

```diff
@@ -146,8 +146,8 @@ extension SQLiteDatabase {
         
         return jobs
     }
-    func getJobs() -> [Job]? {
-        let querySql = "SELECT * FROM job WHERE active == 0 AND failed == '' ORDER BY priority DESC,datetime(created);"
+    
+    func executeAndReturnJobs(querySql: String) -> [Job]? {
```

Comment:
This naming could be missleading. Can you change this to something like getJobsByQuery?